s4e-web/src/app/views/map-view/state/search-results/search-results.service.ts

```diff
@@ -0,0 +1,35 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {SearchResultsStore} from './search-results.store';
+import {PageSearchResult} from './search-result.model';
+import {delay, finalize, map} from 'rxjs/operators';
+import {S4eConfig} from '../../../../utils/initializer/config.service';
+import {configKey, guid} from '@datorama/akita';
+
+@Injectable({providedIn: 'root'})
+export class SearchResultsService {
+
+  constructor(private searchResultsStore: SearchResultsStore,
+              private http: HttpClient,
+              private CONFIG: S4eConfig) {
+  }
+
+  get(placePrefix: string) {
+    this.searchResultsStore.update({isOpen: placePrefix.length > 0});
+    this.searchResultsStore.update({queryString: placePrefix});
```

Comment:
This can be done using one call to `.update`:

```
 this.searchResultsStore.update({isOpen: placePrefix.length > 0, queryString: placePrefix});
```